src/controller/actions/resource.js

```diff
@@ -50,6 +62,28 @@ export const get_resources = id => dispatch => {
     .get(id)
     .then(data => {
       dispatch(resourceSuccess(data));
+      dispatch(setParent(data[0].parent));
+    })
+    .catch(err => {
+      console.log(err);
+      dispatch(resourceError(err));
+    });
+};
+
+/**
+ * Gets 3 most recently viewed resources
+ * First dispatches a loading function to change state to loading
+ * Second: Sends GET request to the server using api.resources
+ * Third: Dispatches resource_success action to add new data from server to current state
+ * If there is an error, it dispatches an error obj to state
+ * *@param {{}}
```

Comment:
This documentation block seems to be a little stale.

Reply:
Set this minor change as an issue or a new task and accept the PR, at the end the important part of this code is ready to go